### PR TITLE
Add logs and vacuum options

### PR DESCRIPTION
**Pull request by tiemay**
Also add note of a couple instances of potentially cleanable json files

### CODE REVIEW

**Comment by tiemay** [comment]
Calibre has a lot of options, many of which I haven't used — I wouldn't be surprised if there where more ways to generate cleanable files.

File: pending/calibre.xml [hunk 1]
```diff
@@ -31,4 +31,21 @@
...
+    <label>Logs</label>
+    <description>Delete the logs</description>
+    <action command="delete" search="file" path="~/.config/calibre/server_access_log.txt"/>
+    <action command="delete" search="file" path="~/.config/calibre/server_error_log.txt"/>
+  </option>
+  <option id="vacuum">
+    <label>Vacuum</label>
+     <description>Clean database fragmentation to reduce space and improve speed without removing any data</description>
+     <!-- NOTE: "~/Calibre Library/" is the default suggested location, however when first setting up Calibre, the user is
+     given the option of choosing a different location - many users may not have their library in the default location -->
```
Comment (az0) [inline pending/calibre.xml]: @tiemay See google_chrome.xml for a JSON example
````action command="json" search="file" path="$localappdata\Google\Chrome\User Data\Default\Preferences" address="dns_prefetching/host_referral_list"````

Comment (tiemay) [inline pending/calibre.xml]: Yea, I saw that, and I tried to use `command="json"` for "gui_view_history" in "~/Calibre Library/metadata_db_prefs_backup.json", but I couldn't get it working.

Comment (az0) [inline pending/calibre.xml]: @tiemay : If you want me to take a look, open a new issue and post that json file (using a link to a gist or pastebin, if necessary).

Comment (tiemay) [inline pending/calibre.xml]: @az0 done, see #91

**Pull request merged**